go.mod

```diff
@@ -71,4 +71,5 @@ replace (
 	// antrea/plugins/octant/go.mod also has this replacement since replace statement in dependencies
 	// were ignored. We need to change antrea/plugins/octant/go.mod if there is any change here.
 	github.com/contiv/ofnet => github.com/wenyingd/ofnet v0.0.0-20210318032909-171b6795a2da
+	github.com/vmware/go-ipfix => github.com/zyiou/go-ipfix v0.0.0-20210503234204-ce70151473b1
```

Comment:
will move to correct version after PR #1949 gets merged because of interface changes in intermediate process. (go-ipfix v0.5.1 does not have `DeleteFlowKeyFromMapWithoutLock` any more, which is useful for flow aggregator implementation in current PR)